src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
+
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/224994214-ad74056c-798d-4c68-865e-f3e0eaa84b07.png)

Reply:
Okay, I got you.
I’m gonna make updates soon

Reply:
Update failures